<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderEditDialog</class>
 <widget class="QDialog" name="OrderEditDialog">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>460</width><height>320</height></rect>
  </property>
  <property name="windowTitle"><string>Заказ</string></property>
  <property name="minimumSize"><size><width>440</width><height>0</height></size></property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="codeLbl">
       <property name="text"><string>Артикул:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="code_cb"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="statusLbl">
       <property name="text"><string>Статус:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="status_cb">
       <item><property name="text"><string>Новый</string></property></item>
       <item><property name="text"><string>В обработке</string></property></item>
       <item><property name="text"><string>Завершен</string></property></item>
       <item><property name="text"><string>Отменён</string></property></item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pointLbl">
       <property name="text"><string>Пункт выдачи:</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="point_cb"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="clientLbl">
       <property name="text"><string>Клиент:</string></property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="client_cb"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="odateLbl">
       <property name="text"><string>Дата заказа:</string></property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="order_date">
       <property name="calendarPopup"><bool>true</bool></property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ddateLbl">
       <property name="text"><string>Дата выдачи:</string></property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="delivery_date">
       <property name="calendarPopup"><bool>true</bool></property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="formSpacer">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>10</height></size></property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <item>
      <spacer name="btnSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text"><string>Сохранить</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text"><string>Отмена</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
